etc/config-template.toml

```diff
@@ -56,6 +56,18 @@
 # max-tasks-per-worker-low = 2000
 # stack-size = "10MB"
 
+[http]
+## If HTTP service is enabled. If the value is true, the Prometheus can pull the metrics directly 
+## from it through the HTTP address. The reason we need it is Prometheus only support HTTP, 
+## but TiKV only supports gRPC and the C gRPC can't support dispatching different protocols.
+# enabled = true
+
+## HTTP listening address.
+# addr = "127.0.0.1:9520". If it is not specified, the system will use it as the default port.
```

Comment:
After commenting out, the config entry becomes:

```
addr = "127.0.0.1:9520". If it is not specified, the system will use it as the default port.
```

And I believe that it is wrong syntax?

Reply:
BTW all configuration items follows the convention that if it is not specified the configuration item in comment will be used as default value. So I think it is not necessary to repeat it here.

Reply:
My bad. @breeswish PTAL.